src/components/Configuration/components.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Popover from 'react-tiny-popover';
+import './Configuration.scss';
+import { timeout } from 'q';
+
+const DEFAULT_CHILDREN_SIZE = 4;
+
+const isFirstChild = child => typeof child === 'string' && child !== ' ';
```

Comment:
I still dont get why we dont do `child.trim() !== ''` instead of doing check for a space, but ok :)